Updated README with proposed process, members, and other info (#8)
HGVSnomenclature/HVNC: README.md

# HGVS Nomenclature

> This repo is the home of the HGVS Nomenclature maintained by the Human Genome Organization Variation Nomenclature Committee.

Reviewer: I would consider putting (HUGO) into the text. People use HUGO a lot and oddly may not always know the full meaning.

## Have Questions?

> Please use the [Discussions](discussions) forum to search for answers and ask new questions.

Reviewer: Do we need a comment to state that some discussions may be closed? Are we in fact going to close discussions once they are acted on. I think just a couple of comments about navigation may help

## Proposal Process Overview

- Anyone may submit a proposal for changes to [Discussions](discussions). The proposal is initially considered OPEN and under discussion.
- Anyone may comment on an OPEN proposal.
- The HVNC will evaluate all proposals from time to time. For each proposal, the committee may choose to:
  - leave the proposal OPEN for further discussion;
  - mark the propossal as REJECTED with explanation if the proposal is deemed unworkable, incomplete, or inconsistent with the existing guidelines, or if it is similar to existing or recently considered proposals; or
  - move the propsal to DRAFTING to indicate that a member of the HVNC will understake drafting of the changes.
- When a proposal is in the DRAFTING stage, a GitHub issue is created to track writing for the intended documentation change. Writing will be undertaken in a branch dedicated for the assigned issue. A pull request will eventually be submitted and evaluated.
- Evaluation of the PR is limited to HVNC members (i.e., not for public discussion), although specific users may be asked for comments when warranted. If the PR is eventually merged, the discussion proposal is marked as ADOPTED.

> - If the PR is not merged for any reason (e.g., if we realize that the PR creates unforeseen issues with the existing nomenclature), the proposal will either be re-opened for more discussion or rejected at the discretion of the committee.

Reviewer: Are we going to have close discussions once complete. Perhaps if we are, come comments about finding and navigating them to help those not used to git

Author: Discussions are open to the public, even with out a github account I believe. So, I think the barrier is about as low as it can be.

## Committee Members

| Name | Term |
|---|---|
| [Johan den Dunnen](https://www.universiteitleiden.nl/en/staffmembers/johan-den-dunnen) (Nederland) | 2013 - |
| [Marina DiStefano](https://www.linkedin.com/in/marina-distefano/) (United States) | 2022 - 2026 |

>  [Peter Freeman](https://research.manchester.ac.uk/en/persons/peter.j.freeman) (United Kingdom) | 2018 - |

Reviewer: Is there a missing pipe character here???

Author: Yep. Thanks!

| [Reece Hart](https://linkedin.com/in/reece) (United States) | 2013 - |
| [Ros Hastings](https://www.researchgate.net/profile/Ros-Hastings)† (United Kingdom) | 2021 - |
| [Jeroen Laros](https://github.com/jfjlaros) (Nederland) | 2022 - 2026 |
| [Terence Murphy]() (United States) | 2021 - 2026 |
| [Rachel Taylor](https://www.linkedin.com/in/rachel-taylor-phd-90a57099/) (United Kingdom) | 2022 - 2026 |
| [Alex Wagner](https://www.linkedin.com/in/alexphd/) (United States) | 2022 - 2026 |

† ISCN representative

### Past Members

- 2013 - 2022
  - Peter Taschner (Nederland)
- 2014 - 2021
  - Anne-Francoise Roux (France)
- 2020 - 2021
  - Oleg Agofonov (Norway)
- 2013 - 2021
  - Donna Maglott (United States)
  - ISCN representative - Jean McGowan-Jordan (United States)
  - HUGO representative - Stylianos Antonarakis (Switzerland)
  - HGVS representative - Marc Greenblatt (United States)
  - HVP / GV representative - Amy McAllister (United Kingdom)
- 2013 - 2018
  - HVP / GV representative - Tim Smith (Australia)
- 2013 - 2014
  - Els Dequeker (Belgium)

> 

Reviewer: Perhaps a section on the wider community who inputs. Ivo Fokkema for example. We may well get regular commitment from certain members of the community so it would be good to credit them.

Author: Great idea. Let's make that another revision so that we can wrap up this PR.

## Reference

den Dunnen JT, Dalgleish R, Maglott DR, Hart RK, Greenblatt MS, McGowan-Jordan J, et al. **HGVS Recommendations for the Description of Sequence Variants: 2016 Update.** Hum Mutat. 2016. [pubmed:26931183](https://www.ncbi.nlm.nih.gov/pubmed/26931183) [doi:10.1002/humu.22981](http://dx.doi.org/10.1002/humu.22981)

## A note about our name

> We've called ourselves by lots of names, including "VarNomen", "MutNomen", "HGVS Recommendations", "HGVS Guidelines", and even just "HGVS" colloquially.  We recognize that this has caused confusion in the community.  Please call us "HGVS Nomenclature".

Reviewer 2: ```suggestion
We've called the nomenclature by lots of names, including "VarNomen", "MutNomen", "HGVS Recommendations", "HGVS Guidelines", and even just "HGVS" colloquially.  We recognize that this has caused confusion in the community.  Please refer to our recommendations as the "HGVS Nomenclature".
```

☝️ proposing this change to differentiate us (the HVNC) from our recommendations (the HGVS Nomenclature).
